music21/noteworthy/translate.py

```diff
@@ -476,33 +528,91 @@ def translateChord(self, attributes):
 
         >>> nwt = noteworthy.translate.NoteworthyTranslator()
         >>> nwt.currentMeasure = measure
-        >>> nwt.translateChord({'Dur': 'Half', 'Pos': '1,3,#5'})
+        >>> nwt.translateChord({'Dur': ['Half'], 'Pos': ['1,3,#5']})
         >>> measure[0]
         <music21.chord.Chord C5 E5 G#5>
 
         Chords also inherit accidentals:
-        >>> nwt.translateChord({'Dur': 'Half', 'Pos': '1,3,5'})
+        >>> nwt.translateChord({'Dur': ['Half'], 'Pos': ['1,3,5']})
         >>> measure[1]
         <music21.chord.Chord C5 E5 G#5>
 
         '''
-        durationInfo = attributes['Dur']
-        pitchInfo = attributes['Pos']
-
-        c = chord.Chord()   # note!
-
-        # durationInfo
-        self.setDurationForObject(c, durationInfo)
-
-        # pitchInfo
-        self.setTieFromPitchInfo(c, pitchInfo)
-        c.pitches = self.getMultiplePitchesFromPositionInfo(pitchInfo)
+        durationInfos = attributes['Dur']
+        pitchInfos = attributes['Pos']
+        currentDuration = self.getCurrentVoice().duration
+        isRestChord = 'Dur2' in attributes
+        i = 0
+
+        def getVoiceAtDuration(inner_self, voiceId, dur):
+            # first check if voice already exists in measure
+            voice = None
+            hasVoice = False
+            for item in inner_self.currentMeasure:
+                if isinstance(item, stream.Voice):
+                    hasVoice = True
+                    if item.id == voiceId:
+                        voice = item
+                        break
```

Comment:
as above, avoid iteration of measure if no voices yet, and in this case you can do

```
voice = None
voice = inner_self.currentMeasure.getElementsByClass(stream.Voice).getElementById(voiceId)
```